xarray/tests/duckarrays/base/reduce.py

```diff
@@ -0,0 +1,144 @@
+import hypothesis.strategies as st
+import numpy as np
+import pytest
+from hypothesis import given, note, settings
+
+from ... import assert_identical
+from . import strategies
+
+
+class VariableReduceTests:
+    def check_reduce(self, obj, op, *args, **kwargs):
+        actual = getattr(obj, op)(*args, **kwargs)
+
+        data = np.asarray(obj.data)
```

Comment:
Could we rename instances of arrays to be called `arr` instead of `data`? It took me a pretty long time to realise that this had nothing to do with hypothesis' `strategies.data`.

Reply:
sure, although it would probably be better to call it `numpy_arr` or move it into the `.copy` call